src/internal/NeoEsp8266UartMethod.h

```diff
@@ -84,7 +83,7 @@ template<typename T_SPEED> class NeoEsp8266UartMethodBase
         CLEAR_PERI_REG_MASK(UART_CONF0(UART1), UART1_INV_MASK);
         SET_PERI_REG_MASK(UART_CONF0(UART1), (BIT(22)));
 
-        _endTime = micros();
+        _startTime = micros() - (getPixelTime() - 50); // After initialization, wait 50 microseconds to force a reset
```

Comment:
Standard UART mode sets TX line to 3.3V while idle. Then we invert the UART logic levels in the line above, to have 0V while idle. This 3.3V to 0V transition can create a bit in the ws2812 bus. We wait 50 microsecond so the bus is reset and that bit is ignored.